README.md

```diff
@@ -39,7 +39,7 @@ var el = {
 ### Converting Javascript Values into Elements
 
 ```javascript
-var minim = require('minim');
+var minim = require('minim').init();
```

Comment:
Couple of questions:

1. Can you have this default export without ES6?
1. If not, and if you make the minim default a new registry, how can you expose the functions for defining your own registries?